bspump/kafka/connection.py

```diff
@@ -35,3 +37,10 @@ class KafkaConnection(Connection):
 
 	def __init__(self, app, id=None, config=None):
 		super().__init__(app, id=id, config=config)
+
+		if self.Config.get("bootstrap_servers") is None:
+			if "kafka" in asab.Config:
+				self.Config.update(asab.Config["kafka"])
+
+		if self.Config.get("bootstrap_servers") is None:
```

Comment:
```
		if self.Config.get("bootstrap_servers") is None:
			if "kafka" in asab.Config:
				self.Config.update(asab.Config["kafka"])
                        else:
                               raise RuntimeError(...)
```

Reply:
What if [kafka] happens to be empty, no url here => this case RuntimeError, won't catch, right?

Reply:
ok :)

Reply:
But bspump config has no default `kafka` section, right? Then L43 would raise `ConfigParser.MissingSectionHeaderError` perhaps.